test/helpers/kubectl.go

```diff
@@ -1401,7 +1403,7 @@ func (kub *Kubectl) Apply(options ApplyOptions) *CmdRes {
 	if len(options.Piped) > 0 {
 		cmd = options.Piped + " | " + cmd
 	}
-	return kub.ExecMiddle(cmd)
+	return kub.Exec(cmd)
```

Comment:
What's the timeout bump here? 30s->4m? This also extends the time to determine a legitimate failure when there's clearly a failure.

Reply:
Maybe an `ExecLongish()` with a 1 min timeout would be better?

Reply:
Would this be cleaner:
```
	ctx, cancel := context.WithTimeout(context.Background(), MidCommandTimeout * 2)
	defer cancel()
	return kub.ExecContext(ctx, cmd)
```